src/lib/sendObservation.js

```diff
@@ -0,0 +1,19 @@
+export async function sendObservation({
+  client,
+  supervisorWebhook,
+  payload,
+  retries = 1
+}) {
+  if (supervisorWebhook) {
+    for (let tries = 0; tries < retries; tries++) {
```

Comment:
Have you met a case where this retry mechanism actually worked?

Reply:
No, but since it preserves the previous functionalities I thought "why not keep it?"

Reply:
No but this is an extra feature so I left it there